tests/op/test_tokenize.py

```diff
@@ -144,3 +146,12 @@ def test_clip_tokens_no_clipping_needed():
     assert result["attention_mask"].shape == (2, 3)
     assert torch.equal(result["input_ids"].to("cpu"), torch.tensor([[1, 2, 3], [4, 5, 6]]))
     assert torch.equal(result["attention_mask"].to("cpu"), torch.tensor([[1, 1, 1], [1, 1, 1]]))
+
+
+def test_tokenize_strings_cpu(model_name="microsoft/deberta-v3-base"):
```

Comment:
Set `microsoft/deberta-v3-base` since that's what Curator's domain and quality classifiers use. LMK if you would prefer it a different way.

Reply:
Nope, looks great to me.